lib/nodes/Operation.ts

```diff
@@ -1,9 +1,5 @@
-/**
- * Created by jpollak on 9/9/16.
- */
+import {Action} from './Action';
 
-import {Base} from './Base';
-
-export class Operation extends Base {
+export class Operation extends Action {
```

Comment:
not sure if this class serves any purpose actually, might be a good idea to remove it?